lib/killbill_client/models/transaction.rb

```diff
@@ -106,7 +106,7 @@ def void(user = nil, reason = nil, comment = nil, options = {}, refresh_options
         end
       end
 
-      def chargeback(user = nil, reason = nil, comment = nil, options = {}, refresh_options = nil)
+      def chargeback(payment_id, user = nil, reason = nil, comment = nil, options = {}, refresh_options = nil)
```

Comment:
No need for the `payment_id` here, since it is taken from the instance variable. Also, let's make sure not to change existing signatures, unless absolutely necessary (this would break Kaui for instance, see https://github.com/killbill/killbill-admin-ui/blob/a5a46e3d8c87146e07eb61cd3f95c8f2e9f52d0d/app/models/kaui/transaction.rb#L19).

Reply:
Sorry for this, I was trying something and I forgot to erase it.